### DISCUSSION TITLE
Work with OpenBB?

### DISCUSSION THREAD

**Discussion by DidierRLopes** [fei-protocol/checkthechain#49] [General]
Hey,

Have you seen OpenBB (https://github.com/OpenBB-finance/OpenBBTerminal) there seems to be some synergies.

Would you be open to working together? 😄

**Comment by sslivkoff** [comment]
hello @DidierRLopes 👋👋

OpenBBTerminal looks like a truly awesome tool

looks like [this](https://github.com/OpenBB-finance/OpenBBTerminal/blob/90467fd322e49c675ae7e4cced09b2cb2ebb3427/data_sources_default.json#L291) is the list of data sources that OpenBB uses for crypto data

one potential synergy here is if OpenBB used ctc as a data source for obtaining on-chain data from decentralized exchanges, decentralized lending markets, and other on-chain DeFi protocols. in this context ctc can be seen as similar to CCXT, but for on-chain data instead of centralized services. aggregators like coin gecko can give high level summaries but a more detailed view of these protocols often requires querying the chain directly

**Reply by DidierRLopes** [reply]
Hey @sslivkoff,

Thanks for the kind words!

Is there any type of data that we are missing on the terminal that you guys have? If so, we could easily integrate you 🙏🏽
